ParkingLotApi/Services/ParkingLotService.cs

```diff
@@ -1,17 +1,24 @@
 ﻿using ParkingLotApi.Dtos;
 using ParkingLotApi.Exceptions;
+using ParkingLotApi.Models;
+using ParkingLotApi.Repositories;
 
 namespace ParkingLotApi.Services
 {
     public class ParkingLotService
     {
-        public async Task<ParkingLotDto> AddAsync(ParkingLotDto parkingLotDto) 
+        private readonly IParkingLotsRepository parkingLotsRepository;
+        public ParkingLotService(IParkingLotsRepository parkingLotsRepository)
+        {
+            this.parkingLotsRepository = parkingLotsRepository;
+        }
+        public async Task<ParkingLot> AddAsync(ParkingLotDto parkingLotDto) 
         { 
             if (parkingLotDto.Capacity < 10)
```

Comment:
verify if the parkingLot is existed can also write here, rather than in repository level